.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  'env': {
+    'browser': true,
```

Comment:
Should probably say node here instead of browser?

Reply:
You are right! It was generated by `yarn eslint --init` but apparently I didn't choose the env properly! 😄 

Thanks for catching it.